### ISSUE TITLE
mystery UI renders app inoperable

### ISSUE THREAD

**Issue by skierpage** [Tommy-Geenexus/exif-eraser#5]
I open the application, "Choose your preferred image source" appears with three badges labeled Image, Image, Images. Tapping them does nothing, press-hold does nothing. Tapping the vertical 6-dot button switches it to double checkmark, tapping again switches it back. The Settings great works but nothing seems relevant. ![Screenshot_20200819-151841](https://user-images.githubusercontent.com/1303668/90695868-1eb97680-e230-11ea-94da-f15384e2a4dc.png)

Last time I used this I somehow managed to get to a dialog to choose an image, but today I'm stuck. Even if something's wonky in my setup you should consider changing your UI.
This on a Pixel 3 running Android 10, build QQ3A.200805.001

**Comment by Tommy-Geenexus** [comment]
There is a launch button underneath the card titles, which is what you are looking for.
Nevertheless, the UI already has been updated for the next version.

**Comment by skierpage** [comment]
> There is a launch button underneath the card titles, which is what you are looking for.

Yes, I figured out the large badge isn't a button, I have to target the little [/]^ icon within the badge to get a list of images.

But there's more confusion. All three Image, Image, and Images [/]^ icons take me to my Downloads directory. That may be where I have the most recent image, but why show it thrice? Also, one time when I clicked  the app displayed a blue bar [ Allow access to "Downloads" ] below the list of "IMAGES ON DOWNLOADS", and clicking that bar displayed a permissions prompt something like
  Allow access to "Downloads" in "Downloads"

(truncated to Downl... in portrait mode), which is odd.

> Nevertheless, the UI already has been updated for the next version.

I'll see if these glitches happen in the new version (I'm running 3.0.3) and if so will report separate bugs.

Thanks, I appreciate your efforts. How do I give you $10 on PayPal or Venmo?
